inventory/environ.py

```diff
@@ -248,9 +248,11 @@ def getDistributedTopology(vars_scope):
     """
     Parse and set parameters to define topology if this is a distributed environment
     """
-    vars_scope["splunk"]["license_master_included"] = bool(os.environ.get("SPLUNK_LICENSE_MASTER_URL"))
+    vars_scope["splunk"]["license_master_url"] = os.environ.get("SPLUNK_LICENSE_MASTER_URL", vars_scope["splunk"].get("license_master_url"))
+    vars_scope["splunk"]["license_master_included"] = bool(vars_scope["splunk"]["license_master_url"])
     vars_scope["splunk"]["deployer_included"] = bool(os.environ.get("SPLUNK_DEPLOYER_URL"))
-    vars_scope["splunk"]["indexer_cluster"] = bool(os.environ.get("SPLUNK_CLUSTER_MASTER_URL"))
+    vars_scope["splunk"]["cluster_master_url"] = os.environ.get("SPLUNK_CLUSTER_MASTER_URL", vars_scope["splunk"].get("cluster_master_url"))
+    vars_scope["splunk"]["indexer_cluster"] = bool(vars_scope["splunk"].get("cluster_master_url"))
     vars_scope["splunk"]["search_head_cluster"] = bool(os.environ.get("SPLUNK_SEARCH_HEAD_CAPTAIN_URL"))
     vars_scope["splunk"]["search_head_cluster_url"] = os.environ.get("SPLUNK_SEARCH_HEAD_CAPTAIN_URL")
```

Comment:
Should lines 257-256 get the same love by defaulting to `os.environ.get("SPLUNK_SEARCH_HEAD_CAPTAIN_URL", vars_scope["splunk"].get("search_head_cluster_url")`?

Reply:
To be consistent, I updated the PR to include support for `splunk.search_head_captain_url` (deprecating the inconsistently-named `splunk.search_head_cluster_url` field which didn't actually work), and also for `splunk.deployer_url`. Like the others, these can be used as alternatives to the SPLUNK_* environment variables, which should still work same as before.